tests/pylint/test_enforce_type_hints.py

```diff
@@ -277,3 +277,69 @@ async def async_get_triggers( #@
 
     with assert_no_messages(linter):
         type_hint_checker.visit_asyncfunctiondef(func_node)
+
+
+def test_invalid_config_flow_step(
+    linter: UnittestLinter, type_hint_checker: BaseChecker
+) -> None:
+    """Ensure invalid hints are rejected for ConfigFlow step."""
+    class_node, func_node, arg_node = astroid.extract_node(
+        """
+    class AxisFlowHandler( #@
+        config_entries.ConfigFlow, domain=AXIS_DOMAIN
+    ):
```

Comment:
For `node.ancestors`, `config_entries.ConfigFlow` is undefined here and thus can't be inferred. Creating a dummy `ConfigFlow` class should work.

Reply:
Done